### PR TITLE
Add the ability to verify a customer source

### PR DESCRIPTION
**Pull request by wbronitsky**
r? @stripe/api-libraries

### CODE REVIEW

File: lib/resources/Customers.js [hunk 1]
```diff
@@ -144,6 +144,12 @@ module.exports = StripeResource.extend({
     urlParams: ['customerId', 'sourceId'],
   }),
 
+  validateSource: stripeMethod({
+    method: 'POST',
+    path: '/{customerId}/sources/{sourceId}/validate',
```
Comment (brandur) [inline lib/resources/Customers.js]: Is this a different URL from your pulls in other languages?

**Comment by brandur** [comment]
Left a question about the API endpoints in use above, but otherwise this looks good to me.

As with PHP, I'll leave this one open a bit longer in case someone else with more Node bindings experience also wants to take a look.

File: lib/resources/Customers.js (hunk 1 above)
Comment (wbronitsky) [inline lib/resources/Customers.js]: shouldn't be. The `customers/` is already taken care of here

Comment (brandur) [inline lib/resources/Customers.js]: Oh I was thinking the `/validate` part at the end (vs. `/verify` elsewhere).

Comment (wbronitsky) [inline lib/resources/Customers.js]: :facepalm: you are totally right. Thanks!

Comment (brandur) [inline lib/resources/Customers.js]: @wbronitsky Cool! And I think the discrepancy might have made it into the function naming as well. Maybe consider a rename from `validateSource` to `verifySource`.

**Comment by brandur** [comment]
Released as 4.1.0.

**Pull request merged**